### DISCUSSION TITLE
Add a search box to plnetwork.io that searches Almanac and other key resources

### DISCUSSION THREAD

**Discussion by bmann** [memser-spaceport/pl-network-hub#14] [Ideas]
Lots of great stuff is documented in the Almanac system at https://protocol.almanac.io/handbook/protocol-labs-spaceport-JzKymu

There are two footer links at plnetwork.io that deep link into Almanac, but it might be useful to highlight that resource more directly, and/or embed a search page at plnetwork.

That way we can promote plnetwork as your default front door, but also quickly get to handbook material that is documented.

**Comment by pandeyanujk** [comment]
@bmann good suggestion! We are planning to reorganize the almanac content & make it more discoverable via the plnetwork.io directly. I will keep you updated on this once we have a timeline.
